Userland/Libraries/LibJS/Tests/builtins/Array/Array.prototype.toReversed.js

```diff
@@ -0,0 +1,12 @@
+describe("normal behavior", () => {
+    test("length is 0", () => {
+        expect(Array.prototype.toReversed).toHaveLength(0);
+    });
+
+    test("basic functionality", () => {
+        const a = [1, 2, 3, 4, 5];
+        const b = a.toReversed();
+        expect(b).toEqual([5, 4, 3, 2, 1]);
+        expect(a).not.toBe(b);
```

Comment:
While this checks that the returned array is not the array that was operated on, it doesn't check that the contents of the original array were unmodified. This is because `toBe` uses `Object.is` internally, which only does a pointer comparison. Could we add a test for that?